teb_local_planner:
  ros__parameters:
    refresh_hz: 20.0
    horizon_length: 8.0
    safety_margin: 0.25
    
    from_bag: false
    measure: false
    
    pre_apex_0: 4.0
    pre_apex_1: 3.0
    pre_apex_2: 1.5
    post_apex_0: 2.0
    post_apex_1: 3.0
    post_apex_2: 4.0
    
    evasion_dist: 0.77
    obs_traj_tresh: 0.3
    spline_bound_mindist: 0.1
    fixed_pred_time: 0.15
    kd_obs_pred: 1.0
    lookahead: 10.0